### ISSUE TITLE
Docs: reference_appendices: replace boolean yes/no with true/false

### ISSUE THREAD

**Issue by samccann** [ansible/ansible#78924]
### Summary

Based on the [steering committee vote to use true/false for booleans,](https://github.com/ansible-community/community-topics/discussions/120) we are going through the guides to adapt to this change.

This issue requests these changes in the `docs/docsite/rst/reference_appendices/` files.

Changes are: change `yes` to `true`  and `no` to `false`

must be lowercase. Please open one PR to handle thse changes. It should impact 7 files. NOTE - ansibot does not like PRs over 50 files.

The following grep will help you find these occurrences from the `docs/docsite/rst/reference_appendices/` directory:
`grep -R '\: \(yes\|no\)$' --exclude-dir=locales`

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/reference_appendices/faq.rst

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
no
```

### OS / Environment

no

### Additional Information

no

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by samccann** [comment]
grep results in the following list:
reference_appendices/general_precedence.rst:     become: yes
reference_appendices/test_strategies.rst:         enabled: yes
reference_appendices/test_strategies.rst:       check_mode: no
reference_appendices/faq.rst:          become: yes
reference_appendices/faq.rst:             remote_src: yes
reference_appendices/YAMLSyntax.rst:    create_key: yes
reference_appendices/YAMLSyntax.rst:    needs_agent: no

**Comment by NiranjanMeghwal** [comment]
Hello, I would like to work on this issue, could you please assign this to me.

**Comment by samccann** [comment]
We don't have the ability to assign in github, but go ahead and create the PR and in the description, say  - Fixes #78924 and it will link back to this issue.

Keep in mind hacktoberfest doesn't start until Saturday, but we probably won't see your PR until Monday. And thanks!
